src/Handler/NewsFeed.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Handler.NewsFeed
+  ( getFeedR
+  ) where
+
+import Import
+
+import qualified Data.Text.Lazy as LT
+import Text.Blaze.Html.Renderer.Text as Text
+
+import qualified Data.Map as M
+
+getFeedR :: Handler TypedContent
+getFeedR = do
+  docs <- getDocs
+  now <- liftIO getCurrentTime
+  tups <- liftIO $ loadAllDocs docs
+  newsFeed $ feed now tups
+
+data LoadedFeedDocs =
+  LoadedFeedDocs
+    { ldocsLibraries :: !(Map Text (Page Html))
+    , ldocsTutorials :: !(Map Text (Page Html))
+    }
+
+loadAllDocs :: Docs -> IO LoadedFeedDocs
+loadAllDocs Docs {..} = do
+  ldocsLibraries <- loadMap docsLibraries
+  ldocsTutorials <- loadMap docsTutorials
+  pure LoadedFeedDocs {..}
+  where
+    loadMap :: Map Text PageHtml -> IO (Map Text (Page Html))
+    loadMap = traverse go
+    go :: PageHtml -> IO (Page Html)
+    go p@Page {..} = do
+      body <- pageBody
+      pure $ p {pageBody = body}
+
+feed :: UTCTime -> LoadedFeedDocs -> Feed (Route App)
+feed now ds =
+  let entries = makeEntries ds
+   in Feed
+        { feedTitle = "FP Complete  Haskell"
+        , feedLinkSelf = FeedR
+        , feedLinkHome = HomeR
+        , feedAuthor = "Tom Sydney Kerckhove <syd@fpcomplete.com"
```

Comment:
I'm not sure if this is intended.

Reply:
Also the closing angle bracket seems to be missing.

Reply:
Which author would we like here?

Reply:
I don't have anything in mind and I don't mind your email being present here! But I thought you might have put it here wrongly instead of an more generic email address!

Reply:
The author should be "FP Complete." This field is for who is the author of the content on the blog, not who coded the feed feature.